loops-and-arrays.js

```diff
@@ -56,6 +58,9 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
-
-
+for (let i = 0; i < squares.length; i++){
+ const volume = squares[i] * squares[i] * squares[i];
```

Comment:
Wist je dat je ook dit kon doen?
```suggestion
 const volume = squares[i] ** 3;
```

Reply:
Nice, thanks!

btw. Hoe kan ik zo een suggested change maken ? :)